protocol/dubbo/dubbo_invoker.go

```diff
@@ -101,8 +101,12 @@ func (di *DubboInvoker) Invoke(ctx context.Context, invocation protocol.Invocati
 		return &result
 	}
 
-	di.AddInvokerTimes(1)
-	defer di.AddInvokerTimes(-1)
+	client := di.getClient()
+	if client == nil {
+		result.Err = protocol.ErrClientClosed
+		logger.Debugf("result.Err: %v", result.Err)
```

Comment:
How about using Warn? I think this only happens in some abnormal cases.

Reply:
good suggestion.